File: wtplayer/src/WTString3D.cpp
#include "basetypes.hpp"
#include "WTString3D.hpp"
#include "WTBitmap.hpp"
#include "WTSurfaceShader.hpp"

void WTString3D::setText(char* text){
  APILOG;
}

void WTString3D::setTextProperties(int drawTextInBold,
                                   int drawTextInItalic,
                                   int quality){
  APILOG;
}

void WTString3D::setColor(int Red,
                          int Green,
                          int Blue,
                          int Alpha){
  APILOG;
}

void WTString3D::setTexture(WTBitmap* pBitmap,
                            int Side){
  APILOG;
}

void WTString3D::setTextureCoordOffset(float U1,
                                       float V1,
                                       int Side){
  APILOG;
}

void WTString3D::setTextureCoordScale(float U1,
                                      float V1,
                                      int Side){
  APILOG;
}

void WTString3D::setSurfaceShader(WTSurfaceShader* pISurfaceShader,
                                  int Side){
  APILOG;
}

void WTString3D::setTextFace(char* fontSearchString,
                             int textCharacterSet){
  APILOG;
}

int WTString3D::getTextLength(){
  APILOG;
  return 0;
}

void WTString3D::setGlyphRelativePosition(int glyphIndex,
                                          float x,
                                          float y,
                                          float Z){
  APILOG;
}

void WTString3D::setGlyphRelativeRotation(int glyphIndex,
                                          float x,
                                          float y,
                                          float Z,
                                          float Angle){
  APILOG;
}

File: wtplayer/javasrc/wildtangent/webdriver/WTString3D.java
package wildtangent.webdriver;

import wildtangent.webdriver.WTBitmap;
import wildtangent.webdriver.WTGroup;
import wildtangent.webdriver.WTSurfaceShader;

public interface WTString3D
extends WTGroup {
    public void setGlyphRelativeRotation(int var1, float var2, float var3, float var4, float var5);

    public void setTextFace(String var1, int var2);

    public void setGlyphRelativePosition(int var1, float var2, float var3, float var4);

    public int getTextLength();

    public void setColor(int var1, int var2, int var3, int var4);

    public void setTextureCoordScale(float var1, float var2, int var3);

    public void setSurfaceShader(WTSurfaceShader var1, int var2);

    public void setTextProperties(int var1, int var2, int var3);

    public void setTextureCoordOffset(float var1, float var2, int var3);

    public void setText(String var1);

    public void setTexture(WTBitmap var1, int var2);
}

